lib/response_comparator.rb

```diff
@@ -25,6 +25,13 @@ def self.quick_comparison(primary_response, secondary_response)
     }
   end
 
+  def self.full_comparison?(comparison, full_pct)
+    r = rand(99)
```

Comment:
Apparently `rand` only returns random numbers between 0 and the argument (not including the argument itself), so perhaps this should be `rand(100)` to capture 0-99